controlplane/kubeadm/main.go

```diff
@@ -54,7 +55,8 @@ func init() {
 
 	_ = clientgoscheme.AddToScheme(scheme)
 	_ = clusterv1.AddToScheme(scheme)
-	_ = kcpv1.AddToScheme(scheme)
+	_ = kubeadmcontrolplanev1old.AddToScheme(scheme)
```

Comment:
```suggestion
	_ = kubeadmcontrolplanev1alpha3.AddToScheme(scheme)
```

Reply:
@vincepri is there any reason why we need to specify api version `v1alpha3` instead of `old`? @fabriziopandini suggest just use `old` to make it easier to change in the future https://github.com/kubernetes-sigs/cluster-api/pull/4417#discussion_r606235766

Reply:
That works for me